ptx/sec_deriv_interpret.ptx

```diff
@@ -1276,20 +1277,22 @@
             </pg-code>
             <statement>
               <image xml:id="img_02_02_ex_17">
-              <shortdescription> f is a hyperbola with curves in first and the third quadrant while the parts of g are in the 3rd and the 4th.</shortdescription>
+              <shortdescription> f is the derivative of g and g is the derivative of f.</shortdescription>
                 <description>
-                <P>
-                The graph of <m>f</m> is made of two curves one in the first and one in the fourth quadrant. Each of these curves has two branches. 
-                One of these two branches extend parallel to the x-axis and the other parallel to the y-axis making a hyperbola. The curve 
-                in the first quadrant has the vertex at <m>(1,1)</m> and the branches of the curve move closer to x-axis and the y-axis, rapidly at
-                first and then close to <m>(4,0)</m> on the x-axis and <m>(0,4)</m> on the y-axis, the branches start to move infinitely 
-                close to the x and y-axis. The second part of <m>f</m> is the mirror image of the first curve that is in the fourth quadrant 
-                with the two branches extending to infinity towardss the negative x and y-axis. <m>g</m> also consists of two curves voth with 
-                two branches. They are in the 3rd and 4th quadrant. The structure of both of these curves are very similar to the two parts of <m>f</m>. 
-                The curve in the 4th quadrant has its focus at <m>(1,-1)</m>.The branch that moves towards negative y-axis moves infinitly close to it as 
-                it continues to infinitly. The branch moving towards x-axis eventually joins the x-axis at <m>(0,4)</m> and continues to infinitly. The 
-                branch in the 4th quadrant is a reflection of the previous branch.
-                </p>
+                <P> The graph of <m>f</m> is a hyperbola with curves in the first and the third quadrant. The curves look like infinite bows with the branches extending in
```

Comment:
You **still** have `<P>` instead of `<p>`!

The book will not build with this typo in place.